### DISCUSSION TITLE
TeX math conversions to docx OMML?

### DISCUSSION THREAD

**Discussion by d-morrison** [jgm/pandoc#10221] [Q&A] [answered]
I'm wondering if anyone has resources for getting complicated TeX math expressions to translate well to OMML (docx/pptx); specifically, colors and arrays:

````quarto
---
format: docx
---

$$\color{red}{x}$$

$$
\begin{array}{l@{\qquad}l}
\text{Infection/colonization episode} & \text{Waning immunity episode}\\
b^{\prime}(t) = \mu_{0}b(t) - cy(t) & b(t) = 0 \\
y^{\prime}(t) = \mu y(t) & y^{\prime}(t) = -\alpha y(t)^r
\end{array}
$$

interestingly, without `{l@{\qquad}l}` the array works:

$$
\begin{array}{}
\text{Infection/colonization episode} & \text{Waning immunity episode}\\
b^{\prime}(t) = \mu_{0}b(t) - cy(t) & b(t) = 0 \\
y^{\prime}(t) = \mu y(t) & y^{\prime}(t) = -\alpha y(t)^r
\end{array}
$$
````

**Comment by jgm** [comment, marked as answer]
For the array issue, you can submit a bug report at jgm/texmath.  (Btw, what does the `@{\qquad}` do here?)

Similarly for colors, separate issue please, but it's much less likely to be supported in the near term because it would require changes to the texmath types and all the readers and writers.  Perhaps for now we could simply ignore colors.

**Reply by d-morrison** [reply]
Thanks for your reply! `@{\qquad}` changes the width of the between-column space:

![image](https://github.com/user-attachments/assets/e46ddf3d-f2c9-4134-b61b-07d98ef0c228)

I guess it's also a good candidate for ignoring?

**Reply by d-morrison** [reply]
(posted issues #10222 and #10223)
